scripts/bundles/plugins/alias-plugin.ts

```diff
@@ -50,6 +50,9 @@ export function aliasPlugin(opts: BuildOptions): Plugin {
       if (id === '@utils') {
         return join(opts.buildDir, 'utils', 'index.js');
       }
+      if (id === '@utils/shadow-css') {
+        return join(opts.buildDir, 'utils', 'shadow-css.js');
+      }
```

Comment:
Nice improvement 👌 instead of having two `shadow-css.js` files in `internal/client/shadow-css.js` and `rollup/hydrate/shadow-css.js` it is now only one in `internal/client/shadow-css.js`, imported by `internal/hydrate/index.js` via `await import("../client/shadow-css.js")`

Reply:
yeah exactly, previously it was getting duplicated because different Rollup bundles were causing it to be copied over several times